package.json

```diff
@@ -25,9 +25,10 @@
     "@reactioncommerce/db-version-check": "^1.0.0",
     "@reactioncommerce/file-collections": "~0.7.0",
     "@reactioncommerce/file-collections-sa-gridfs": "~0.1.0",
-    "@reactioncommerce/plugin-simple-authorization": "~1.0.0",
     "@reactioncommerce/logger": "~1.1.2",
     "@reactioncommerce/nodemailer": "~5.0.5",
+    "@reactioncommerce/plugin-simple-authorization": "~1.0.0",
+    "@reactioncommerce/plugin-system-information": "^1.0.0",
```

Comment:
is there any reason this particular plugin uses the `^` for its version. I believe the `~` should be used so that only patch versions are updated automatically.

Reply:
No reason, when I `npm install`ed this is what it automatically did, I just didn't update. Changed now.